en/docs/edit-the-code/code-actions.md

```diff
@@ -1 +1,70 @@
-# Code actions
\ No newline at end of file
+# Code actions
+
+## Types of Code actions
+
+### QuickFixes
+
+QuickFixes are suggested based on the diagnostic displayed at a given cursor position.
+
+#### Code Actions for Create Variables
```

Comment:
Shall we change this to `Quickfixes for variable assignment required` diagnostic. And move `Create variable with type` to a separate section like `Quickfixes for type inference`?